documentation/docs/getting-started/installation.md

```diff
@@ -376,10 +376,10 @@ You can change your LLM provider and/or model or update your API key at any time
 
 ## Shared Configuration Settings
 
-The Goose CLI and Desktop UI share all core configurations, including LLM provider settings, model selection, and extension configurations. When you install or configure extensions in either interface, the settings are stored in a central location at `~/.config/goose/config.yaml`, making them available to both the Desktop application and CLI. This makes it convenient to switch between interfaces while maintaining consistent settings.
+The Goose CLI and Desktop UI share all core configurations, including LLM provider settings, model selection, and extension configurations. When you install or configure extensions in either interface, the settings are stored in a central location at `~/.config/goose/`, making them available to both the Desktop application and CLI. This makes it convenient to switch between interfaces while maintaining consistent settings. For more information, visit the [Config Files][config-files] guide.
```

Comment:
To avoid confusion, since the Windows path will be different, should we just say `... the settings are stored in a central location, making them available ...` ?